lib/game.rb

```diff
@@ -85,14 +86,34 @@ def is_column_full?(input)
     @board.visual[input].none?(".")
   end
 
-  def win?
-    if sw_to_ne_diagonal_win? || se_to_nw_diagonal_win? || horizontal_win? || vertical_win?
-      'True'
-    else
-      'False'
+  def game_over
```

Comment:
We'll add some sweet ASCII art when the winner is declared, but nice job on getting the game_over method to check if a win condition was met OR if the game resulted in a draw